src/codeManager.ts

```diff
@@ -162,7 +168,17 @@ export class CodeManager {
         fs.writeFileSync(this._codeFile, content);
     }
 
-    private getExecutor(languageId: string, fileExtension: string): string {
+    private async getSubExecutor(executor: any): Promise<string> {
+        let subExecutorId = await vscode.window.showQuickPick(Object.keys(executor), { placeHolder: "Type or select command to run" });
+        if (subExecutorId !== undefined) {
+            executor = executor[subExecutorId];
+        } else {
+            executor = null;

```

Comment:
Just `return null` is not good experience for users. It will lead to `vscode.window.showInformationMessage('Code language not supported or defined.');` be called.